docs/long_desc.rst

```diff
@@ -3,11 +3,33 @@ BanditPAM: A state-of-the-art, high-performance k-medoids algorithm
 
 Introduction
 ------------
-Clustering algorithms such as k-means are ubiquitous in modern data science applications. Despite its popularity, however, k-means has several drawbacks. K-means only supports certain distance metrics, and its cluster centers can lack interpretability. Alternatively, k-medoids uses the points in the dataset itself -- the medoids -- as cluster centers. This enables interpretability of the cluster centers. Furthermore, k-medoids supports arbitrary dissimilarity functions in place of a distance metric.
+Clustering algorithms such as k-means are ubiquitous in modern data science applications. 
+Despite its popularity, however, k-means has several drawbacks. Firstly, k-means only supports certain distance metrics, 
+while different distance metrics may be desirable in other applications. For example, L1 norm and cosine distance are often used in
+recommendation systems and single-cell RNA-seq analysis. Secondly, the cluster center of k-means is in general not a point 
+in the dataset and hence lack of interpretability. This is problemetic when the problem requires interpretability on the center. 
+For example, when the data is structured, such as images in computer vision where the mean image is virtually random noise.
```

Comment:
I think we should move this after the description of k-medoids. We want to emphasize k-medoids and our contributions first.

Reply:
Also, `visually random noise`